src/lib/types.ts

```diff
@@ -231,6 +231,7 @@ export type CommonHeroProps = {
   title: string
   description: string
   buttons?: [CallToActionProps, CallToActionProps?]
+  dir?: "rtl" | "ltr"
```

Comment:
I think this is also no longer needed

Reply:
Hm... I had added this to HubHero props, but I'm not actually using it yet... I believe this was in preparation for updating the other page directions using `contentNotTranslated` logic, but that will be in a separate PR. Will revert this and the HubHero prop changes for now.